bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -503,6 +561,7 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <!-- ================================================================== -->
 
 <z0param_method>ZengWang2007</z0param_method>
+<z0param_method phys="clm6_0">Meier2022</z0param_method>
 <z0param_method phys="clm5_1">Meier2022</z0param_method>
```

Comment:
Hmmm. I'll look to see if I can use this pattern to reduce the instances of setting all physics options. This pattern only works if there is only two options used. Then you set one option by default to the option most commonly used, and only set the ones that diverge from that. Otherwise we have to set for: clm4_5, clm5_0, clm5_1, and clm6_0 so four options. So you can save a line in some cases by doing this pattern.

I also wondered if consistency was more important, and if setting each option was more readable.  But, this is something we should be on the same page as a group with.

Reply:
OK, I've done this and think it's a lot better.

Reply:
@slevis-lmwg let me know if you think this is improved, and if so go ahead and check this as approved from your angle. Thanks...